### PR TITLE
plan: move explan_test.go to plan package.

### PR DESCRIPTION
**Pull request by hanfei1991**
will increase coverage to 82%

### CODE REVIEW

**Comment by hanfei1991** [comment]
@zz-jason PTAL

**Comment by zz-jason** [comment]
LGTM

File: plan/explain_test.go [hunk 1]
```diff
@@ -11,18 +11,24 @@
...
+var _ = Suite(&testExplainSuite{})
+
+type testExplainSuite struct {
+}
+
+func (s *testExplainSuite) TestExplain(c *C) {
+	store, err := newStoreWithBootstrap()
+	c.Assert(err, IsNil)
+	tk := testkit.NewTestKit(c, store)
 	defer func() {
```
Comment (tiancaiamao) [inline plan/explain_test.go]: I guess you should `defer store.Close()`, and `defer domain.Close()`, otherwise testleak won't happy.

Comment (tiancaiamao) [inline plan/explain_test.go]: `make leak` to verify that.

**Pull request merged**